### DISCUSSION TITLE
또다시 올리는 초안 글...

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#602] [4. 기타 글 모음]
안녕하세요~ 이번주에도 글을 완성해서 포스팅 하긴 어려울 것 같아서 초안 글들을 업로드하게 되었습니다 😭
이번주에 정말 중요한 일정이 하나 있었는데요...!! 이후로 되게 많은 생각을 하게 되었습니다..... 얼른 회고를 포스팅하고 싶지만, 지금은 때가 아닌 것 같네요 🙂 나중에 올리도록 하겠습니다~

제가 포스팅한 마지막 글이 7월 4일자 글인데, 최근 바쁜 일상을 보내고 있다 보니 글 쓸 시간이 없었던게 너무 아쉽네요...
사실 저는 어느정도 이런 생활에 익숙해져 있었지만, 저번주~이번주에는 진짜 핵폭탄을 맞은 느낌이라 글쓰기가 힘들었습니다 😅

작성해야 할 초안 글들은 다음과 같아요
- [구주소 검색 초안 글](https://shkisme.notion.site/ElasticSearch-5-2915ad135163431e83a7abf0eb11efbe?pvs=4)
- 스프링 인터셉터를 활용한 HTTP 요청 & 응답 전역 로깅 구현기
- 오픈서치 테스트 환경 구축 (가능하다면)

다음주에는!! 꼭! 완성된 글을 하나 들고오도록 노력하겠습니다~~ 화이팅!

**Comment by 02ggang9** [comment]
화이팅입니다~~
